content/en/docs/v3.1/op-guide/configuration.md

```diff
@@ -283,13 +283,13 @@ Follow the instructions when using these flags.
 + Set level of detail for exported metrics, specify 'extensive' to include histogram metrics.
 + default: basic
 
-[build-cluster]: clustering#static
-[discovery]: clustering#discovery
+[build-cluster]: ../clustering#static
```

Comment:
Missing a trailing `/` here, and on the line below.

Pls also double check other pages for missing trailing slashes.

Reply:
Btw, I'm only mentioning these (relative links and trailing slashes) because yesterday you mentioned that you make a pass over the PR to ensure conformance to the guidelines.

If you'd rather move on to another section, say v3.2, and stick to the guidelines there, as you make the link updates, I'm ok with that too.